spec/functional/core/core.server.spec.js

```diff
@@ -0,0 +1,11 @@
+var request = require("supertest"),
+    app = require("server");
+describe("Server : ", function() {
+    describe("Server should return appropraite server response.", function() {
+        it("should response to /", function(done) {
+            request(app).get('/').expect('Content-Type', /json/).expect(200).end(function(err, res) {
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'res' is defined but never used.